### PR TITLE
Fix tiltrotor model and add to SITL testing

### PR DESCRIPTION
**Pull request by Jaeyoung-Lim**
This PR fixes the tiltrotor model that was failing SITL testing before, and adds the tests to mavsdk testing

### CODE REVIEW

File: models/tiltrotor/tiltrotor.sdf [hunk 1]
```diff
@@ -1035,9 +1028,15 @@
       <accelerometerBiasCorrelationTime>300.0</accelerometerBiasCorrelationTime>
       <accelerometerTurnOnBiasSigma>0.196</accelerometerTurnOnBiasSigma>
     </plugin>
-    <plugin name='groundtruth_plugin' filename='libgazebo_groundtruth_plugin.so'>
-      <robotNamespace/>
-    </plugin>
+    <include>
```
Comment (TSC21) [inline models/tiltrotor/tiltrotor.sdf]: I would actually prefer this where it was since here it's like in the middle of the plugins.

Comment (Jaeyoung-Lim) [reply models/tiltrotor/tiltrotor.sdf]: Done

**Comment by BazookaJoe1900** [comment]
I am not sure, but think that this should be changed:
https://github.com/PX4/sitl_gazebo/blob/1f49d1a915e27a19aeebd674a453c24ea7f463b2/models/tiltrotor/tiltrotor.sdf.jinja#L712


to
`<pose>0 0.6 0 0.00 0 0.0</pose> `

**Comment by Jaeyoung-Lim** [comment]
@BazookaJoe1900 Could you clarify why?

**Comment by BazookaJoe1900** [comment]
because of the symmetry to the right_elevon:
https://github.com/PX4/sitl_gazebo/blob/d9e2527cc91d3a2e32b01b8b7c671f6a2dfd0075/models/tiltrotor/tiltrotor.sdf.jinja#L741

**Comment by Jaeyoung-Lim** [comment]
Closing since there are too many intermittent failures for this model

Will reopen when there is time to revisit

**Pull request closed without merging**